templates/python3/files/.platform.template.yaml

```diff
@@ -39,7 +39,7 @@ info:
 # This key describes the initialization call made to the master environment at
 # project creation time. This is part of the full v2 UI operation mode, which
 # places project schema/options selection early in the creation process, rather
-# than later as it exitss now. To allow this schema to be backwards-compatible,
+# than later as it exits now. To allow this schema to be backwards-compatible,
```

Comment:
"exists" :-)

For a follow-up: "as it exists now" is no longer true since the new UI is deployed, so we probably want to fix this block across the board at some point.